### ISSUE TITLE
Radio-group component does not apply aria-label correctly

### ISSUE THREAD

**Issue by jefftmahoney** [IBM/carbon-components-angular#966]
## Detailed description

> Describe in detail the issue you're having.
A value I supplied for the `aria-label` attribute is not being placed correctly within the template of the `ibm-radio-group` component.

> Is this a feature request (new component, new icon), a bug, or a general issue?
A bug.

> Is this issue related to a specific component?
Yes - `ibm-radio-group`

> What did you expect to happen? 
Basically, I expected the value I supplied for the `aria-label` attribute to be applied by the component to its own template in such a way that it would not cause problems when running an accessibility-compliance tool like the DAP plugin for Chrome.  

> What happened instead?
What happened instead is that DAP complained that a `<div>` with a `role` specified with "radiogroup" did not have a corresponding `aria-label` attribute (this div is defined within the template of the `ibm-radio-group` component).  

> What would you like to see changed?
It'd be nice if the `ibm-radio-group` had an angular input for aria-label (with a binding to this input defined in the template) so that whatever value was supplied would get placed there and everybody, especially DAP, would be happy.

> What browser are you working in?
Chrome

> What version of the Carbon Design System are you using?
10.6.0 for `carbon-components` and 3.21.2 for `carbon-components-angular`

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
IBM Study Advance (in Watson Health).  Our team would like to pass accessibility for our product by late January

## Steps to reproduce the issue

This code will generate the output shown in the screenshot (attached)
```
      <ibm-radio-group [(ngModel)]="objective.objectiveType" (ngModelChange)="objectiveObjectiveTypeChanged()">
        <div class="radio-button-spacer">
          <ibm-radio [checked]="objective.objectiveType === 'primary'" [value]="'primary'" [disabled]="readOnly" i18n>
            Primary
          </ibm-radio>
        </div>
        <div class="radio-button-spacer">
          <ibm-radio [checked]="objective.objectiveType === 'secondary'" [value]="'secondary'" [disabled]="readOnly"
            i18n>
            Secondary
          </ibm-radio>
        </div>
        <div class="radio-button-spacer">
          <ibm-radio [checked]="objective.objectiveType === 'exploratory'" [value]="'exploratory'" [disabled]="readOnly"
            i18n>
            Exploratory
          </ibm-radio>
        </div>
      </ibm-radio-group>
```
## Additional information

* Screenshots or code
Note how the highlighted `<div>` does not have an `aria-label` attribute applied to it.  This is the problem that DAP has with it.  I think we just need to have the component apply `aria-label` as an attribute to that div (with whatever value the developer specified), and then we'll be good.
![radio-group-aria-issue](https://user-images.githubusercontent.com/2761266/71029578-a8330a80-20dd-11ea-94e7-0c28f610205c.png)

* Notes

**Comment by Donisius** [comment]
Thank you so much for pointing this out!
